controllers/controllers/services/bindings/managed/controller.go

```diff
@@ -273,9 +252,41 @@ func (r *ManagedBindingsReconciler) finalizeCFServiceBinding(
 }
 
 func (r *ManagedBindingsReconciler) reconcileSBServiceBinding(ctx context.Context, cfServiceBinding *korifiv1alpha1.CFServiceBinding) (*servicebindingv1beta1.ServiceBinding, error) {
+	log := logr.FromContextOrDiscard(ctx)
+
+	secretLookupKey := types.NamespacedName{Name: cfServiceBinding.Name, Namespace: cfServiceBinding.Namespace}
+	createdSecret := new(corev1.Secret)
```

Comment:
I would suggest renaming `createdSecret` to `credentialsSecret` as the secret here stores the credentials for the binding. This name would convey what that secret is (improves readability)

I would also change the way the secret is being looked up to be align with the way we lookup objects:

```golang
credentialsSecret := &corev1.Secret{
	ObjectMeta: metav1.ObjectMeta{
			Name:      cfServiceBinding.Name,
			Namespace: cfServiceBinding.Namespace,
		},
}
if err := r.k8sClient.Get(ctx, client.ObjectKeyFromObject(credentialsSecret), credentialsSecret); err != nil {
	return nil, err
}
```